src/jenkinsNextTest/groovy/hudson/plugins/gradle/AbstractIntegrationTest.groovy

```diff
@@ -0,0 +1,34 @@
+package hudson.plugins.gradle
+
+import hudson.model.Cause
+import hudson.model.FreeStyleBuild
+import hudson.model.FreeStyleProject
+import hudson.model.ParametersAction
+import hudson.model.ParametersDefinitionProperty
+import hudson.model.TextParameterDefinition
+import hudson.model.TextParameterValue
+import hudson.model.queue.QueueTaskFuture
+import org.junit.Rule
+import org.junit.rules.RuleChain
+import org.jvnet.hudson.test.JenkinsRule
+import spock.lang.Specification
+
+class AbstractIntegrationTest extends Specification {
```

Comment:
That class it now duplicated - on `jenkinsNextTest` and `test`.

Reply:
Yes. I didn't find a way around it, given that the AST transformation from the different Spock version seem to be different enough that when I try to move `AbstractIntegrationTest` to `testFixtures`, the `jenkinsNextTests`s break.